core/templates/dev/head/pages/error/error.html

```diff
@@ -67,6 +67,9 @@ <h2 class="oppia-error-h2" translate="I18N_ERROR_MESSAGE_500"></h2>
     .oppia-footer {
       position: absolute;
     }
+    .oppia-wide-panel-content {
+      margin-bottom: 240px;
```

Comment:
@llKCll `margin-bottom: 240px` is inheriting in the responsive views. And, please do check the code with all responsive views before pushing the commit.